components/admin_console/plugin_management/plugin_management.tsx

```diff
@@ -165,7 +181,7 @@ const PluginItem = ({
         activateButton = (
             <a
                 data-plugin-id={pluginStatus.id}
-                disabled={deactivating}
+                data-disabled={deactivating}
```

Comment:
Why were these changed?

Reply:
These were added because of the ts error `Type '{ children: Element; data-plugin-id: string; disabled: boolean; onClick: (e: any) => any; }' is not assignable to type 'DetailedHTMLProps<AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement>'.
  Property 'disabled' does not exist on type 'DetailedHTMLProps<AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement>'.ts(2322)`

Reply:
It looks like `disabled` doesn't do anything on anchor tags, so we can actually just remove this line entirely. `data-disabled` would only be relevant if some code was consuming it.